src/__integrations__/testCases/retryingFetch.ts

```diff
@@ -0,0 +1,18 @@
+import { Millisecond } from 'italia-ts-commons/lib/units';
+import { fetch as polyfilledFetch } from 'whatwg-fetch';
+import { retryingFetch } from '../../utils/fetch';
+
+function retryingFetchTest() {
+  // If the environment doesn't have an abort controller load a polyfill
+  if (!globalThis.AbortController) {
+    require('abortcontroller-polyfill/dist/abortcontroller-polyfill-only');
```

Comment:
note that abortcontroller polyfill is already imported here:

https://github.com/pagopa/io-ts-commons/blob/657a12490d336d226e13b8ab997759c6a9f23e6e/src/fetch.ts#L17

Reply:
Thank you for pointing out. I'll try to have a single abort controller polyfill and a unique import point